pkg/config/system_probe.go

```diff
@@ -172,6 +172,7 @@ func InitSystemProbeConfig(cfg Config) {
 
 	cfg.BindEnvAndSetDefault(join(smNS, "enable_java_tls_support"), false)
 	cfg.BindEnvAndSetDefault(join(smNS, "java_agent_args"), defaultServiceMonitoringJavaAgentArgs)
+	cfg.BindEnvAndSetDefault(join(smNS, "enable_http_status_code_aggregation"), false)
```

Comment:
I'd suggest renaming this. IMHO enabling aggregation feels like it should result in less stats, not more.

Reply:
How about `enable_http_status_code_collection`?